irr/include/IMesh.h

```diff
@@ -52,6 +52,9 @@ enum E_ANIMATED_MESH_TYPE
 	//! Halflife MDL model file
 	EAMT_MDL_HALFLIFE,
 
+	//! Graphics Language Transmission Format 2.0 (.gltf) mesh
+	EAMT_GLTF2,
```

Comment:
I don't think we need this yet; the loader will just produce a skinned mesh until we support more features.

Reply:
I thought it should be listed here because all the other formats seemed to be, but maybe all the other formats have their own special mesh implementations. I agree with just using skinned mesh until there's a good reason to use something else.

Reply:
pretty sure this should be skinned mesh in any case, because existing code may check this value:
```
irr/src/CAnimatedMeshSceneNode.cpp
161:	if (Mesh->getMeshType() != EAMT_SKINNED) {
248:			else if (Mesh->getMeshType() == EAMT_SKINNED)
275:				else if (Mesh->getMeshType() == EAMT_SKINNED)
294:				if (Mesh->getMeshType() == EAMT_SKINNED)
302:			if (Mesh->getMeshType() == EAMT_SKINNED) {
328:				else if (Mesh->getMeshType() == EAMT_SKINNED)
404:	if (!Mesh || Mesh->getMeshType() != EAMT_SKINNED) {
432:	if (!Mesh || Mesh->getMeshType() != EAMT_SKINNED) {
450:	if (!Mesh || Mesh->getMeshType() != EAMT_SKINNED)
599:	if (Mesh && Mesh->getMeshType() == EAMT_SKINNED) {
669:	if (!Mesh || Mesh->getMeshType() != EAMT_SKINNED)
```